anaron/03.CSS/06_CSS_Adaptation.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<HTML>
+
+<HEAD>
+    <meta charset="utf-8">
+    <TITLE>Proba</TITLE>
+    <style type="text/css">
+        ul {
+            list-style-type: none;
+            margin: 0;
+            padding: 0;
+            border-radius: 5px;
+            border: 1px solid black;
+            border-bottom: none;
+            display: inline-block;
+            width: 120px;
+            height: 25px;
+        }
+
+        li {
+            position: relative;
+        }
+
+        li ul {
+            display: none;
+            position: absolute;
+            top: 26px;

```

Comment:
here better to use auto value instead 26px,

Reply:
fixed) Thank you)